deep_conmech/simulator/setting/setting_matrices.py

```diff
@@ -1,11 +1,6 @@
-from typing import Callable
-
 import deep_conmech.common.config as config
```

Comment:
I think config should be imported only in one file and then injected to others

Reply:
Agreed, but it will be a bigger job - we have to unify configuration throughout deep_conmech and conmech

Reply:
So can you create issue for that?